### PR TITLE
Fix part of #973 : Fix app module tests for Roboelectric in ExplorationActivityTest 

### PR DESCRIPTION
**Pull request by anandwana001**
## Explanation

Fix part of #973
Fix for ExplorationActivityTest
Ignored are part of 89

### Espresso
Screenshot 2020-09-17 at 09 05 47

### Robolectric
Screenshot 2020-09-17 at 09 01 22

## Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [x] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

### CODE REVIEW

File: app/src/sharedTest/java/org/oppia/app/player/exploration/ExplorationActivityTest.kt
Comment (rt4914) [inline app/src/sharedTest/java/org/oppia/app/player/exploration/ExplorationActivityTest.kt:288]: Try to remove `//ktlint` check everywhere.
One solution can be:
`testAudioWithNoVoiceover_openPrototypeExp_configChange_audioButtonNotVisible`

File: app/src/sharedTest/java/org/oppia/app/player/exploration/ExplorationActivityTest.kt
Comment (rt4914) [inline app/src/sharedTest/java/org/oppia/app/player/exploration/ExplorationActivityTest.kt:788]: There is warning in this line have a look.

File: app/src/sharedTest/java/org/oppia/app/player/exploration/ExplorationActivityTest.kt
Comment (rt4914) [inline app/src/sharedTest/java/org/oppia/app/player/exploration/ExplorationActivityTest.kt]: Nit `return ApplicationProvider.getApplicationContext<TestApplication>().isOnRobolectric()`

**Review by rt4914 — APPROVED** [review]
This PR is really nice. Thanks a lot. Will wait for @vinitamurthi 's approval and nit changes.

File: app/src/sharedTest/java/org/oppia/app/player/exploration/ExplorationActivityTest.kt
Comment (vinitamurthi) [inline app/src/sharedTest/java/org/oppia/app/player/exploration/ExplorationActivityTest.kt]: Do we need this import? If we are reusing something from StateFragmentTest, lets move that part to the testing module and have StateFragmentTest and this test use that

**Review by vinitamurthi — APPROVED** [review]
Just one comment, otherwise LGTM

File: app/src/sharedTest/java/org/oppia/app/player/exploration/ExplorationActivityTest.kt
Comment (anandwana001) [reply app/src/sharedTest/java/org/oppia/app/player/exploration/ExplorationActivityTest.kt]: Done

File: app/src/sharedTest/java/org/oppia/app/player/exploration/ExplorationActivityTest.kt
Comment (anandwana001) [reply app/src/sharedTest/java/org/oppia/app/player/exploration/ExplorationActivityTest.kt:788]: Done

File: app/src/sharedTest/java/org/oppia/app/player/exploration/ExplorationActivityTest.kt
Comment (anandwana001) [reply app/src/sharedTest/java/org/oppia/app/player/exploration/ExplorationActivityTest.kt]: Good catch, Thanks

File: app/src/sharedTest/java/org/oppia/app/player/exploration/ExplorationActivityTest.kt
Comment (anandwana001) [reply app/src/sharedTest/java/org/oppia/app/player/exploration/ExplorationActivityTest.kt:288]: It seems like a bit long issue to resolve and it's almost in all the files. I will create a good first issue for this.

File: app/src/sharedTest/java/org/oppia/app/player/exploration/ExplorationActivityTest.kt
Comment (vinitamurthi) [reply app/src/sharedTest/java/org/oppia/app/player/exploration/ExplorationActivityTest.kt]: LGTM now, resolving this

File: app/src/sharedTest/java/org/oppia/app/player/exploration/ExplorationActivityTest.kt
Comment (aggarwalpulkit596) [reply app/src/sharedTest/java/org/oppia/app/player/exploration/ExplorationActivityTest.kt:788]: Use named arguments instead ?

Comment (aggarwalpulkit596) [reply app/src/sharedTest/java/org/oppia/app/player/exploration/ExplorationActivityTest.kt:788]: Here and elsewhere too

File: app/src/sharedTest/java/org/oppia/app/player/exploration/ExplorationActivityTest.kt
Comment (anandwana001) [reply app/src/sharedTest/java/org/oppia/app/player/exploration/ExplorationActivityTest.kt:288]: https://github.com/oppia/oppia-android/issues/1855

File: app/src/sharedTest/java/org/oppia/app/player/exploration/ExplorationActivityTest.kt
Comment (anandwana001) [reply app/src/sharedTest/java/org/oppia/app/player/exploration/ExplorationActivityTest.kt:788]: It seems like, named argument is not supported here as invoke is a java function

**Review by rt4914 — APPROVED** [review]
LGTM, thanks.

**Pull request merged**
